### DISCUSSION TITLE
What in the name of Zeus is a custom ndll

### DISCUSSION THREAD

**Discussion by UberfireF-5** [CodenameCrew/CodenameEngine#362] [Q&A] [answered]
I know this sounds like a self-answering question, but I don't know what this is supposed to do. Does it go into the addons folder, or does it go into a mod's folder? What is the purpose of this feature?

**Comment by NeeEoo** [comment, marked as answer]
It allows mods to run custom c++ code without the need of a custom exe file. It enables the possibilities of Transparent Windows, Cool Fourth Wall Stuffs, Sending Notifications, and more

**Reply by NexIsDumb** [reply]
even lua support

**Reply by UberfireF-5** [reply]
Ohhhhhhh. That makes sense. Thanks for the info.
